ir/base.def

```diff
@@ -120,6 +120,7 @@ abstract Type_Base : Type {
 class Type_Unknown : Type_Base {
 #nodbprint
     static Type_Unknown get();
+    static Type_Unknown get(const Util::SourceInfo &si);
```

Comment:
Is there ever a case where we have an unknown type with a source info?  I can't think of how it could occur.

Reply:
I do not think so. At least we do not store any source info for an unknown type when parsing. I mostly mirrored what the auto-generated constructors allow, so I do not think at can hurt at least.